### DISCUSSION TITLE
[Feature] Landing Page Setting

### DISCUSSION THREAD

**Discussion by clembuck** [immich-app/immich#12569] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello,

It would be great to have the option to select which tab users initially land on.

**Use case scenario:**
Some users may have very few or no personal photos and primarily connect to view others' shared pictures. It would be helpful for these users to be able to land directly on the Albums tab, where shared albums are displayed.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by Dezc** [comment]
I second this !

**Comment by svcx650** [comment]
I would add to this, when I used Google photos I longed for the ability to open to my camera roll/photos on my device rather than all photos. I would love to see that option implemented on immich.
